tests/e2e/e2e_test.go

```diff
@@ -1303,9 +1304,22 @@ func (s *IntegrationTestSuite) PacketForwarding() {
 	// check the balance of the contract
 	s.CallCheckBalance(chainANode, contractAddr, "uosmo", transferAmount)
 
+	// Getting the sender as set by PFM
+	fmt.Println("Getting the sender as set by PFM")
+	fmt.Println("validatorAddr", validatorAddr)
+	senderStr := fmt.Sprintf("channel-0/%s", validatorAddr)
+	senderHash32 := address.Hash(packetforwardingtypes.ModuleName, []byte(senderStr)) // typo intended
+	sender := sdk.AccAddress(senderHash32[:20])
+	bech32Prefix := "osmo"
+	pfmSender, err := sdk.Bech32ifyAddressBytes(bech32Prefix, sender)
+	s.Require().NoError(err)
+	fmt.Println("PFM sender", pfmSender)
+
 	// sender wasm addr
-	senderBech32, err := ibchookskeeper.DeriveIntermediateSender("channel-0", validatorAddr, "osmo")
+	senderBech32, err := ibchookskeeper.DeriveIntermediateSender("channel-0", pfmSender, "osmo")
+	fmt.Println("senderBech32", senderBech32)
```

Comment:
If prints are only meant for local debugging, let's remove them please

Reply:
removed